omnibus_overrides.rb

```diff
@@ -5,7 +5,7 @@
 override :erlang, version: "18.3.4.9"
 override :lua, version: "5.1.5"
 override :rubygems, version: "3.0.3"
-override :bundler, version: "1.17.2" # currently pinned to what ships in Ruby to prevent double bundler
+override :bundler, version: "1.17.3" # currently pinned to what ships in Ruby to prevent double bundler
```

Comment:
```suggestion
override :bundler, version: "1.17.3"
```

Reply:
I like having the comment in there, just so that our future selves know easily what that change was made, but that can be put as the commit message perhaps?